documentation/modules/exploit/windows/local/canon_driver_privesc.md

```diff
@@ -0,0 +1,92 @@
+## Vulnerable Application
+
+Canon TR150 print drivers versions 3.71.2.10 and below allow local users to read/write files
+within the `CanonBJ` directory and its subdirectories. By writing a malicious DLL to the
+`C:\\ProgramData\\CanonBJ\\IJPrinter\\CNMWINDOWS\\Canon TR150 series\\LanguageModules\\040C\\CNMurGE.dll`
```

Comment:
technically we overwrite this DLL, might want to reword this.

Reply:
Fixed wording in https://github.com/rapid7/metasploit-framework/pull/15520/commits/55404ff29feb0e68e143778f20fe520c72243633